docs/api/graph/line-styles.md

```diff
@@ -4,6 +4,7 @@ There are four [line styles](../../asl/ref/ls.md) available:
 * [Area](#area)
 * [Stack](#stack)
 * [Vertical Span](#vertical-span)
+* [Heat Map](#heat-map)
```

Comment:
I think we should be consistent and use it as one word everywhere in the docs (`Heatmap`).